spec/index.html

```diff
@@ -167,57 +167,71 @@ <h3>Resources and Statements</h3>
       exists, without explicitly naming it.</p>
   </section>
 
-  <section id="section-quoted-triples">
-    <h3>Quoted Triples</h3>
+  <section id="section-graph-terms">
+    <h3>Graph Terms</h3>
 
-    <p>A <a>quoted triple</a> is an <a>RDF term</a> with the components of
-      an <a>RDF triple</a>, which can be used as the <a>subject</a> or <a>object</a>
+    <p>A <a>graph term</a> is an <a>RDF term</a> with the components of
+      an <a>RDF graph</a>, which can be used as the <a>subject</a> or <a>object</a>
       of another triple.</p>
 
-    <p>A <a>quoted triple</a> is not necessarily asserted, allowing
+    <p>A <a>graph term</a> is not necessarily asserted, allowing
       statements to be made about other statements that may not be
-      asserted within an <a>RDF graph</a>.
+      asserted within another <a>RDF graph</a>.
       This allows statements to be made about relationships that may be contradictory.
-      For a <a>quoted triple</a> to be asserted,
-      it must also appear in a graph as an <a>asserted triple</a>.</p>
+      For a <a>graph term</a> to be asserted,
+      the triples it contains must also appear in a graph which is not a <a>graph term</a>.</p>
```

Comment:
Based on the same rationale as [my previous change suggestion](https://github.com/w3c/rdf-concepts/pull/67/files#r1361930313), I suggest to change this sentence as follows.
```suggestion
      For the triples in a <a>graph term</a> to be asserted,
      they must also appear in a graph which is not a <a>graph term</a>.</p>
```